FOLLOW-UP.md

```diff
@@ -4,18 +4,16 @@
 
```

Comment:
I think we should get rid of the follow up entirely. It's an extra part that is covered later in the recruitment process (rational for technical choices, depth of expertise etc).

Reply:
Maybe we shrink it to not cover so many requirements to get some understanding on choice made e.g. why not Backpack over other libs used (could be due to docs or something else) or choices on adding libs that may not be necessary, afterall its something we grade on and would be good to get test feedback.

Can always trial having it and then remove it if the answers aren't giving much extra value and we can give a guide of approx 10 mins to write a quick follow up on the test overall